.eslintrc.js

```diff
@@ -1,18 +1,186 @@
+require('dotenv-flow').config({
+  silent: true,
+});
```

Comment:
what's the story here? were we not respecting .env files properly before this?

Reply:
This was included in the eslint config we were using previously; most of the contents in the .eslintrc.js here are just copied from the existing config.